packages/core/strapi/src/commands/actions/plugin/verify/command.ts

```diff
@@ -0,0 +1,17 @@
+import type { StrapiCommand } from '../../../types';
+import { runAction } from '../../../utils/helpers';
+import action from './action';
+
+/**
+ * `$ strapi plugin:verify`
+ */
+const command: StrapiCommand = ({ command, ctx }) => {
+  command
+    .command('plugin:verify')
+    .description('Verify the output of your plugin before publishing it.')
+    .option('-d, --debug', 'Enable debugging mode with verbose logs', false)
+    .option('--silent', "Don't log anything", false)
```

Comment:
Just wondering if these make sense in the context of verify? I'm thinking whether it would make sense to log `--debug` automatically when verification fails.

On the contrary I am not sure what to expect from `--silent`?

Reply:
RE. silent, it's consistent with other commands & those args typically extend round the strapi CLI (albeit perhaps not fully implemented yet), and you _might_ not want the logs and just to know it fails. I'd say the main reason would be consistency.

> would make sense to log --debug automatically when verification fails.

How would you do that? Most times we print error messages and account for everything we can the debug just prints additional info mainly used if you get an unexpected error only. But equally if it failed how could we retroactively print all the logs? I suppose you could store all your debug logs but not print them if the flag isnt there 🤔 Is that what you were thinking?